massgen/cli/quickstart.py

```
@@ -0,0 +1,208 @@
+
+def _pull_docker_image_headless() -> bool:
+    """Pull default Docker image without interactive prompts.
+
+    Returns:
+        True if image was pulled successfully, False otherwise.
+    """
+    import subprocess
+
+    image = "ghcr.io/massgen/mcp-runtime-sudo:latest"
+    try:
+        subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)